homeassistant/components/tami4/__init__.py

```diff
@@ -0,0 +1,37 @@
+"""The Tami4Edge integration."""
+from __future__ import annotations
+
+from Tami4EdgeAPI import Tami4EdgeAPI
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import Platform
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import ConfigEntryNotReady
+
+from .const import CONF_REFRESH_TOKEN, DOMAIN
+
+PLATFORMS: list[Platform] = [Platform.SENSOR]
+
+
+async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
+    """Set up edge from a config entry."""
+    refresh_token = entry.data.get(CONF_REFRESH_TOKEN)
+
+    try:
+        edge = await hass.async_add_executor_job(Tami4EdgeAPI, refresh_token)
```

Comment:
```suggestion
        api = await hass.async_add_executor_job(Tami4EdgeAPI, refresh_token)
```

In my opinion, the word `api` describes better the object as `edge`